homework_11/homework_11.py

```diff
@@ -0,0 +1,72 @@
+# 1 Написати власний декоратор, задачею якого має бути друк назви функції і
+# часу, коли вона була викликана. Декоратор має працювати для різних функцій однаково.
+
+import time
+
+
+def deco_time_and_name(func_to_deco):
+    def print_time_and_name(*args, **kwargs):
+        str_name = func_to_deco.__name__
+        print(f"function name - {str_name}")
+        str_time = time.ctime()
+        print(f"start function time - {str_time}")
+        func_to_deco()
```

Comment:
Pass *args and **kwargs here. This means that you pass arguments to the function you are decorating. 
And add return statement for the result for func_to_deco call but now a separate one
`return func_to_deco(*args, **kwargs)`

Reply:
The return is still missing